src/CoreBundle/Resources/public/javascript/tableEditor.js

```diff
@@ -710,14 +706,99 @@ CHAMELEON.CORE.MTTableEditor.DeleteRecordWithCustomConfirmMessage = function (sC
     if (confirm(sConfirmText)) {
         window.onbeforeunload = function () {
         };
-        PleaseWait();
+        CHAMELEON.CORE.processingDialog('show');
         document.cmseditform.elements['module_fnc[contentmodule]'].value = 'Delete';
         document.cmseditform.submit();
     }
-}
+};
+
+$(document).ready(function () {
+    CHAMELEON.CORE.MTTableEditor.setGetActiveTab();
+    CHAMELEON.CORE.MTTableEditor.inputFields();
+});
 
-function updateIframeSize(sFieldName,iHeight){
+
+CHAMELEON.CORE.MTTableEditor.setGetActiveTab = function () {
+    var url = document.URL;
+    var hash = url.substring(url.indexOf('#'));
+
+    $(".nav-tabs").find("li a").each(function (key, val) {
+        if (hash == $(val).attr('href')) {
```

Comment:
saved the jquery element to a variable
We can't get rid of the loop, because it does 2 things. set the active tab matching the current hash and adding events for all link elements.
So using a selector would add more code instead if reducing it and would need to call the selector twice.